pincer/objects/guild/guild.py

```diff
@@ -1538,6 +1545,105 @@ async def delete_template(
             construct_client_dict(self._client, data)
         )
 
+    @classmethod
+    async def sticker_packs(cls) -> Generator[StickerPack, None, None]:
```

Comment:
If we are going to have more of the `Generator[..., None, None]` typing, it would be nice to have a custom type alias

Reply:
I looked it up and I believe that `Generator[YieldType, SendType, ReturnType]` can be replaced with `Iterator[YieldType]`.
Like this:
```suggestion
    async def sticker_packs(cls) -> Iterator[StickerPack]:
```